content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio is configured by default to capture all traces. For every time
+you access `/productpage` you should see a corresponding trace in the
+Jaeger dashboard. This is suitable for a test or low traffic mesh. For
+a high traffic mesh you may wish to lower the trace sampling
```

Comment:
a high traffic mesh, you can lower the trace sampling

Reply:
Updated.